sections/changes.include

```diff
@@ -82,9 +84,9 @@
   <dt><a href="https://github.com/w3c/html/commit/49073d89a7829dc3dbbfc3802f38bf9845902117">Remove references to Polyglot Markup</a></dt>
   <dd>Removes references to the Polyglot Markup specification (now published as a W3C Note).</dd>
   <dt><a href="https://github.com/w3c/html/commit/0b959a586e54fe4145b69303ebb11755cea26c09">Make menu and menuitem match reality</a></dt>
-  <dd>Removes the radio and checkbox types for menus; requires that <{menuitem}> has content; <{menuitem}> makes a command if it is labelled; changes terminology from "popup menu" state to "context menu" state.</dd> 
+  <dd>Removes the radio and checkbox types for menus; requires that <{menuitem}> has content; <{menuitem}> makes a command if it is labelled; changes terminology from "popup menu" state to "context menu" state.</dd>
```

Comment:
(This could use a linebreak too while you're here...)

Reply:
Not sure where you want a linebreak...? Looks the same as the other sections to me.